React/Modules/RCTAnimationDriverNode.m

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTAnimationDriverNode.h"
+#import <UIKit/UIKit.h>
+#import "RCTAnimation.h"
+
+const double SINGLE_FRAME_INTERVAL = 1.f / 60.f;
+
+@interface RCTAnimationDriverNode ()
+
+@property (nonatomic, copy) RCTResponseSenderBlock callback;
+
+@end
+
+@implementation RCTAnimationDriverNode {
+  NSArray *_frames;
+  double _delay;
```

Comment:
Prefer NSTimeInterval for timings. It's just an alias for double anyway, but it makes it clear that the timing is in seconds.

Aside: On the JS side we generally use milliseconds (as is conventional) for timings and on the iOS side we generally use seconds (made clear by using NSTimeInterval). If you ever deviate from that, please document it heavily with inline comments, or even consider naming the properties with an "MS" or  "Seconds" suffix.